system-tests/driver-config/jenkins.sh

```diff
@@ -29,7 +29,7 @@ targets:
       PATH: "/usr/local/bin:$PATH"
 
     scripts:
-      proxy: http-server --proxy-secure=false -p 4201 -P \$CLUSTER_URL dist
+      proxy: http-server --proxy-secure=false -p 4201 -P \$CLUSTER_URL ../../dist
```

Comment:
We talked offline, the Jenkins driver is always ensuring that the path's in this file are relative to the file location. So it will always hit the right directory. Unless we move the file, I can live with this drawback.